### DISCUSSION TITLE
Why does my script run automatically？

### DISCUSSION THREAD

**Discussion by guhao-myob** [dbeaver/dbeaver#10693] [Q&A] [answered]
#### System information: 
- MacOs Catalina 10.15.7
- DBeaver community  7.3.0

#### Connection specification:
- Mysql 5.7 

#### Describe your question:

This is my procedure：
1. The program generates sql statements
2. I posted the sql statement to the running interface and did nothing
3. Click to run the sql script
4. The mistake is a duplicate entry
5. Check the execution log and find that there are two, one for success and one for failure.

I am not sure when the successful one was executed, because the two were too close, and it seems to only happen to me. I tried to paste sql on another machine, but this did not happen.

**Comment by kseniiaguzeeva** [comment]
Could you please give a query example?

**Comment by guhao-myob** [comment]
> Could you please give a query example?

As shown below, it's just a simple insert, but more than 7000 rows need to be inserted, the sql will be a bit more than the following.

```
insert into dbname.tablename (id,name,email,password)
values(1,1,1,1),(2,2,2,2)....(7000,7000,7000,7000)
```


Just now I think I found the reason
When I click 'read data in sql console', 
It will automatically execute this default query statement. When my table is very large and it has not yet returned, when I switch the default query statement to my own SQL statement, it will become the execution of my SQL statement.

**Comment by guhao-myob** [comment, marked as answer]
> Could you please give a query example?

I think i should new a SQL editor to run it instead of  'read data in sql console' . 
🧎‍♀️
